integration-tests/features/steps/server_api.py

```diff
@@ -38,3 +51,8 @@ def check_versions(context, num=0, versions='', ecosystem='', package=''):
         assert v['ecosystem'] == ecosystem
         assert v['package'] == package
         assert v['version'] in versions
+
+
+@then('I should get 400 status code')
+def check_error_code(context):
```

Comment:
Please note that we need to document all public methods and functions.

Reply:
Additionally, the prerequisity 'I should get 400 status code' is already covered by implemented test step.

Reply:
We don't need this function ;)